src/main.js

```diff
@@ -60,36 +60,23 @@ function newRandomCover() {
 }
 
 function newUserCover() {
-  document.onload.preventDefault();
+  event.preventDefault();
```

Comment:
Good work on this event.preventDefault(); update! This seemed to fix the issue on my end!